app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Name.java

```diff
@@ -0,0 +1,57 @@
+package uk.co.ribot.androidboilerplate.data.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Name implements Parcelable {
+    public String first;
+    public String last;
+
+    public Name() {
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Name name = (Name) o;
+
+        if (first != null ? !first.equals(name.first) : name.first != null) return false;
+        return !(last != null ? !last.equals(name.last) : name.last != null);
+
```

Comment:
Android studio automatically generates that bit of code with that space, I'm not sure why. I don't think it matters too much... Same for the one below.